tests/test_playbooks/upload.yml

```diff
@@ -16,6 +16,16 @@
   - include: tasks/repository.yml
     vars:
       repository_state: present
+  - include: tasks/repository.yml
+    vars:
+      repository_state: absent
+      repository_name: "Test Deb Repository"
+      repository_content_type: "deb"
```

Comment:
technically not needed here

Reply:
right, but otherwise the content_type would be defaulted to yum by that task, and that feels wrong.

Reply:
yet you don't specify it on  line 88+ where the main destroy happens :D

Reply:
Now we do.